test/e2e/suite/command/list.go

```diff
@@ -50,4 +50,24 @@ var _ = Describe("notation list", func() {
 				NoMatchErrKeyWords(HTTPSRequest)
 		})
 	})
+
+	It("all signatures of an oci-layout", func() {
+		HostWithOCILayout(BaseOptionsWithExperimental(), func(notation *utils.ExecOpts, ociLayout *OCILayout, vhost *utils.VirtualHost) {
+			notation.Exec("sign", "--oci-layout", ociLayout.ReferenceWithDigest()).
+				MatchKeyWords(SignSuccessfully)
+
+			notation.Exec("list", "--oci-layout", ociLayout.ReferenceWithDigest()).
+				MatchKeyWords(
+					"└── application/vnd.cncf.notary.signature",
+					"└── sha256:",
```

Comment:
Can we match the digest instead of just `sha256`. On success, the signature generated by `sign` should be listed out.

Reply:
Updated to use a signed oci-layout.